### ISSUE TITLE
Command doesn't show the commant line parameters 

### ISSUE THREAD

**Issue by Randl** [Syllo/nvtop#12]
I see `python3` instead if `python3 myscript.py --resume --gpu` which makes it harder to identify processes.

**Comment by Syllo** [comment]
Unfortunately the Nvidia interface only returns the process name.

I will finish the pretty old patch that searches the process name and arguments from the ``/proc/pid/cmdline`` when available.

**Comment by Syllo** [comment]
Can you please try the patch present in the ``process_names`` branch if it resolves your problem?

**Comment by Randl** [comment]
@Syllo yes, this resolves the problem. For the long commands the ability to scroll as in htop would be nice, but probably I should open a separate issue for that?
